fence-edit.el

```diff
@@ -167,8 +167,10 @@ edit."
   (interactive)
   (save-excursion
     (goto-char pos)
-    (forward-line)
-    (line-beginning-position)))
+    (if (not (eq (following-char) 10))
+	(point)
```

Comment:
For best compatibility with other text display directionality, you should probably still use `line-beginning-position` here.

Reply:
But `line-beginning-position` would cause the narrowed edit to include the ends which I'm seeking to avoid with this change. For example given 
```
(add-to-list 'fence-edit-blocks
               '("\\(:\n\\|^\\)\\s-*\"\"\"" "\\(:\n\\|^\\)\\s-*\"\"\"" rst)
```
when I call `fence-edit-code-at-point` with my point in the following text:
```
"""Lorem ipsum dolor sit amet, consectetur adipiscing elit. 

Aliquam ut convallis sem.
"""
```
I want the narrowed edit to contain:
```
Lorem ipsum dolor sit amet, consectetur adipiscing elit. 

Aliquam ut convallis sem.
```
However I believe using `line-beginning-position` here causes it to include `"""` at the beginning like this:
```
"""Lorem ipsum dolor sit amet, consectetur adipiscing elit. 

Aliquam ut convallis sem.
```